src/functions/durableOrchestration.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpRequest, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableOrchestrationClient, IOrchestrationFunctionContext } from 'durable-functions/lib/src/classes';
+
+df.orchestration('DurableFunctionsOrchestratorJS', function* (context: IOrchestrationFunctionContext) {
+    const outputs = [];
+    outputs.push(yield context.df.callActivity('Hello', 'Tokyo'));
```

Comment:
Make a `const` for the function name `Hello` and reference it here. IMO it's one of the main benefits of putting all 3 in one file and serves as a fix for the comment you removed from the template:
> // Replace "Hello" with the name of your Durable Activity Function.

Reply:
Done